src/NuGet.Clients/Options/PackageSourcesOptionsControl.cs

```diff
@@ -95,15 +95,22 @@ private void UpdateUI()
             var selectedSource = (Configuration.PackageSource)PackageSourcesListBox.SelectedItem;
             var selectedMachineSource = (Configuration.PackageSource)MachineWidePackageSourcesListBox.SelectedItem;
 
-            if (selectedSource != null)
+            if (selectedMachineSource != null)
+            {
+                // THIS BLOCK corresponds to MachineWidePackageSourcesListBox
+
+                addButton.Enabled = removeButton.Enabled = MoveUpButton.Enabled = MoveDownButton.Enabled = BrowseButton.Enabled = updateButton.Enabled = false;
+
+                NewPackageName.ReadOnly = NewPackageSource.ReadOnly = true;
+            }
+            else
             {
                 // THIS BLOCK corresponds to PackageSourcesListBox
 
                 MoveUpButton.Enabled = selectedSource != null && PackageSourcesListBox.SelectedIndex > 0;
                 MoveDownButton.Enabled = selectedSource != null && PackageSourcesListBox.SelectedIndex < PackageSourcesListBox.Items.Count - 1;
 
-                // do not allow deleting the official NuGet source
-                bool allowEditing = selectedSource != null && !selectedSource.IsOfficial;
+                bool allowEditing = selectedSource != null;
```

Comment:
Yes, it's intentional and it's unrelated to the issue, this is just old design, it's already changed, so did clean up here